app/src/pages/inside/profilePage/configExamplesBlock/tabsConfig.jsx

```diff
@@ -19,7 +19,7 @@ const javaConfig = (token, activeProject) => ({
       <div className={cx('options')}>
         <p>rp.enable = true</p>
         <p>rp.description = My awesome launch</p>
-        <p>rp.tags = TAG1;TAG2</p>
+        <p>rp.attributes = TAG1;TAG2</p>
```

Comment:
Please update it for all configuration examples. Also please clarify with QA team how we should show default values for this examples. Now attributes are array with the next structure:
```
{
key: string,
value: string,
system: boolean
}
```